_data/retail.yml

```diff
@@ -1080,6 +1080,20 @@ websites:
   bch: true
   btc: true
   othercrypto: false
+- name: Monorean
+  url: https://www.monorean.com/
+  img: monorean.png
+  twitter: monorean
+  facebook: monorean
+  bch: Yes
+  btc: No
+  othercrypto: No
+  doc: http://blog.monorean.com/en/bitcoin-cash-accepted-here
```

Comment:
I’m on my mobile but is this able to work as HTTPS?

Also, are we sure they don’t also accept BTC? We like to be accurate about listings.

This does look like an exciting new merchant!

Reply:
Just tested and they do not have HTtPS for the blog section. No need to change this then